2_observations.yml

```diff
@@ -0,0 +1,22 @@
+target_default: 2_observations
+
+packages:
+  - readr
+  - zip
+
+sources:
+  - src/file_functions.R
+
+targets:
+  2_observations:
+    depends:
+      - out_data/temperature_observations.zip
+      - out_data/flow_observations.zip
+
+  # daily flow and temperature data
+  out_data/temperature_observations.zip:
```

Comment:
One place of caution is to figure out if you need to lock any of these targets down, or you want them to rebuild every time they change in the other pipeline. Depends on if the pipeline if serving other needs at the same time while you are working through this data release or not.

Reply:
Good thought. How have you handled that? Put it in `data_in` and disconnect from the pipeline that builds it?

Reply:
for this, you could add a `dummy` argument to zip_obs that contains the date as a string, and then you could put the file name in `I()` so that changes to the actual file wouldn't be noticed by remake (only changes to the actual file name or changes to the `dummy` date would trigger a rebuild. That approach would allow you to still make it clear that the file comes from that other pipeline, but locking it into a version on a certain date. Or you could put it in `in_data` and add some documentation that says where it came from. I think I prefer the `dummy` solution, but I've only run into this once (and was bitten by it) in our lake temp pipeline, since that file was serving several concurrent projects at once.